### ISSUE TITLE
PhotoJS fails to load environment variables defined in the prisma schema

### ISSUE THREAD

**Issue by THPubs** [prisma/prisma2#956]
Now we can set environment variables inside the prisma schema file as defined in here: https://github.com/prisma/prisma2/blob/master/docs/prisma-schema-file.md#using-environment-variables . When we run CLI commands, it will automatically load the values from the `.env` file. But, when we use photon js, it won't pick it up. To fix this, we have to use `dotenv` right before instantiating photon like this:

```javascript
import { Photon } from '@generated/photon'
import * as dotenv from 'dotenv';

dotenv.config(); // Load the environment variables
const photon = new Photon()
```

Now there are two ways to fix this:

- We can mention this in the documentation.
- We can call `dotenv` inside the photonjs library itself.

What do you think is the best? My vote is for the second option. I can create a PR the best choice.

**Comment by pantharshit00** [comment]
I can confirm this as a bug. We should load the env file if it is present and right now photon is not loading the .env file.

**Comment by pantharshit00** [comment]
The portion in the docs
![image](https://user-images.githubusercontent.com/22195362/69349195-60b38d00-0c9d-11ea-94e0-39ea56ca9452.png)

**Comment by MikeyUchiha** [comment]
@janpio @pantharshit00 will this be in the Preview 18? environment variables are really important for security reasons.

**Comment by janpio** [comment]
Probably not, work on Preview18 is already in progress (and there is more than enough to get through anyway).

If I understood @pantharshit00 correctly you can use actual environment variables just fine, it's just the ones defined in a .env file that are not picked up. The initial issue message also provided a workaround.

**Comment by pantharshit00** [comment]
Yes, you can use something like direnv(https://github.com/direnv/direnv) to manage environment variables in the meanwhile as a workaround.

**Comment by divyenduz** [comment]
@THPubs This behavior is expected, I created an issue to spec this behavior (why we made this design decision) and another one to have the docs better reflect this. Thank you.
